docs/variants/pc_engines/releases_seabios.md

```diff
@@ -0,0 +1,19 @@
+# Release Notes
+
+Following Release Notes describe status of Dasharo SeaBIOS variant of
+open-source firmware development for PC Engines apu2 series platform.
+
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+<center>
+[Subscribe to PC Engines apu2 series Dasharo SeaBIOS Release Newsletter]
+[newsletter]{.md-button .md-button--primary .center}
+</center>
+
+Test results for this platform can be found
+[here](TBD).
+
+COMING SOON
+
+[newsletter]: TBD
```

Comment:
There is a need for DES information here:
- Dasharo Entry Subscription for Network Appliance
- Dasharo Entry Subscription for Network Appliance Legacy - this will consist both mainline and legacy

Reply:
https://github.com/Dasharo/docs/pull/794/commits/d88e1db523898f4842a0cd51f25a9bff02b99563

Reply:
I also have created empty mailtrain lists with above names: 0105203528f403b0a60bb3755f42ea95f3e6ed54

Reply:
`Dasharo Entry Subscription for Network Appliance Newsletter Legacy` this sounds very weird; Dasharo flavour would be Dasharo Entry Subscription for Network Appliance Legacy, so it should be `Dasharo Entry Subscription for Network Appliance Legacy Newsletter`

Reply:
fixed in https://github.com/Dasharo/docs/pull/794/commits/0105203528f403b0a60bb3755f42ea95f3e6ed54